src/Control/Lens/Cons.hs

```diff
@@ -319,6 +323,42 @@ _tail :: Cons s s a a => Traversal' s s
 _tail = _Cons._2
 {-# INLINE _tail #-}
 
+-- | Convert one type with a 'Cons' instance into the other.
+--
+-- Rewrite rules are provided for efficient conversion between
+-- 'String' and 'Text', @['Word8']@ and 'ByteString', and lazy and
+-- strict 'Text' and 'ByteString'.  Programs must be compiled
+-- with @-O@ to use them.
```

Comment:
This text is misleading, you can understand that you can convert between all of those. AFAICS you can convert `ByteString` to `[Word8]` and `Text` to `[Char]`, and lazy to strict variants. How this is better than `strict` / `lazy` and `packed` / `unpacked` ?